### PR TITLE
release v0.1.0

### PR DESCRIPTION
**Pull request by shyam-patel-kira**

### CODE REVIEW

**Review by Markkos89 — APPROVED** [review]

File: packages/nextjs/components/EventCard.tsx [hunk 1]
```diff
@@ -32,16 +32,26 @@ export const EventCard = ({
...
+        >
+          ¡Reclama esta Badge! 🎖️
+        </Link>
+      ) : null}
+      {!(Number(eventDetails.startTimestamp) * 1000 < Date.now()) ? (
+        <Link
+          href={`/event/${eventDetails?.eventId}/register`}
+          className="text-xl text-white absolute inset-0 flex items-center justify-center opacity-0 group-hover:opacity-100 transition-opacity duration-300 bg-black/50 rounded-md backdrop-blur-md"
+        >
+          ¡Registrarse al evento!
```
Comment (HabacucMX) [inline packages/nextjs/components/EventCard.tsx:53]: This is not working properly, im still able to register for the event even when it is finalized:

https://github.com/user-attachments/assets/fb72d241-6085-4ae7-9e95-7bd34aa11dbc

**Review by HabacucMX — CHANGES_REQUESTED** [review]

File: packages/nextjs/app/event/[eventId]/register/page.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        setShowWarningToast(false);
+      }, 4000);
+      () => clearTimeout(timeout);
+    }
+  };
+  return (
+    <div className="mx-auto">
+      <form onSubmit={handleSubmit} className="rounded-md bg-gray-300/80 p-4 w-96 grid gap-4 ">
+        <div className="text-2xl font-mus text-center">
+          Registro a {/* Event_Name */}:
```
Comment (HabacucMX) [inline packages/nextjs/app/event/[eventId]/register/page.tsx]: @Markkos89 please add the `Event Name` and the `Event Date` in the registry titles

File: packages/nextjs/components/EventCard.tsx (hunk 1 above)
Comment (Markkos89) [reply packages/nextjs/components/EventCard.tsx:53]: fixed on #45 

cc: @HabacucMX

File: packages/nextjs/app/event/[eventId]/register/page.tsx (hunk 2 above)
Comment (Markkos89) [reply packages/nextjs/app/event/[eventId]/register/page.tsx]: Fixed on #45 

cc: @HabacucMX

**Comment by Markkos89** [comment]
> Pending changes:
> 
> * [ ]  restrict access to the event registration after the event is finalized
> * [ ]  Translate and add event context in Register to event
> * [ ]  enable markdown format to Event Detail (event_id)

Fixed item 1 and 3 on #45 

@HabacucMX can you explain what do you mean with `Translate and add event context in Register to event` ? 
What is the event `context`?
What needs to be translated?

**Review by HabacucMX — APPROVED** [review]

**Pull request merged**